dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4742,7 +4740,7 @@ def test_read_parquet_convert_string_nullable_mapper(tmp_path, engine):
         ddf = dd.read_parquet(
             tmp_path,
             engine="pyarrow",
-            use_nullable_dtypes="pandas",
+            use_nullable_dtypes="numpy_nullable",
```

Comment:
This is supposed to be a boolean, not a string

```suggestion
            use_nullable_dtypes=True,
```